<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageColorModelWidget</class>
 <widget class="QWidget" name="ImageColorModelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="hLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wartość H</string>
   </property>
  </widget>
  <widget class="QLabel" name="sLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wartość S</string>
   </property>
  </widget>
  <widget class="QLabel" name="vLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>220</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wartość V</string>
   </property>
  </widget>
  <widget class="QLabel" name="vValue">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>220</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QSlider" name="hValueSlider">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>140</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-359</number>
   </property>
   <property name="maximum">
    <number>359</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="vValueSlider">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>240</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="ImageColorModelCanvas" name="colorModelCanvas" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>551</width>
     <height>561</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="sValueSlider">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>190</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="hValue">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="sValue">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageColorModelCanvas</class>
   <extends>QWidget</extends>
   <header>imagecolormodelcanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
